src/migmose/parsing.py

```diff
@@ -0,0 +1,85 @@
+"""
+contains functions for file handling and parsing.
+"""
+
+import json
+from pathlib import Path
+from typing import Generator, Union
+
+import click
+import docx  # type: ignore[import]
+from docx.document import Document  # type: ignore[import]
+from docx.oxml import CT_Tbl  # type: ignore[import]
+from docx.table import Table, _Cell  # type: ignore[import]
+from docx.text.paragraph import Paragraph  # type: ignore[import]
+from loguru import logger
+from maus.edifact import EdifactFormat
+
+
+def find_file_to_type(message_types: list[EdifactFormat], input_dir: Path) -> dict[EdifactFormat, Path]:
```

Comment:
statt `type` würde ich lieber bei dem fachlichen wording bleiben. Also eher `format` oder `EdifactFormat`
Das dann in dem Funktionsnamen und dem parameter `message_type` anpassen.

Reply:
109bcb4bec527ff08fa10eee0c7266a28907f9ac